homeassistant/components/zwave_me/config_flow.py

```diff
@@ -0,0 +1,88 @@
+"""Config flow to configure ZWaveMe integration."""
+
+from dataclasses import asdict
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import CONF_TOKEN, CONF_URL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZWaveMeConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """ZWaveMe integration config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Initialize flow."""
+        self.url = vol.UNDEFINED
+        self.token = vol.UNDEFINED
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        errors = {}
+
+        if self._async_current_entries():
```

Comment:
Why limit to a single instance?

Reply:
We were not sure if HA allows multiple instances. We have not found how to reasonable allow multiple instances in config_flow.py

Reply:
User can start a new config flow and enter a new URL. It will just work. You do need to make sure you don't store things in `hass.data[DOMAIN]` but in `hass.data[DOMAIN][entry.entry_id]`